### PR TITLE
Chore: (Docs) Updates the loader docs

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the Loaders docs are updated and polished a bit to convey a clearer picture.

What was done:
- Polished the documentation a bit.
- Included examples for React (MDX), Vue 3 (JS and MDX), Angular (TS and MDX), Svelte (JS and MDX).

A small caveat to this, the Vue 2 snippets are still not working and will require a bit of research, the same goes with Svelte native format.

Feel free to provide feedback

### CODE REVIEW

File: docs/snippets/common/storybook-preview-global-loader.js.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import fetch from 'node-fetch';
 
 export const loaders = [
   async () => ({
-    currentUser: (await fetch('https://jsonplaceholder.typicode.com/users/1')).json(),
+    currentUser: await (await fetch('https://jsonplaceholder.typicode.com/todos/1')).json(),
```
Comment (shilman) [inline docs/snippets/common/storybook-preview-global-loader.js.mdx:8]: shouldn't this be fetching a user?

File: docs/snippets/svelte/loader-story.js.mdx [hunk 2]
```diff
@@ -5,14 +5,19 @@ import fetch from 'node-fetch';
...
+};
+
 export const Primary = (args, { loaded: { todo } }) => ({
   Component: TodoItem,
-  props: {...args, ...todo},
+  props: { ...args, ...todo },
 });
 
-Primary.loaders = [
+Primaryloaders = [
```
Comment (shilman) [inline docs/snippets/svelte/loader-story.js.mdx:18]: typo?

File: docs/snippets/vue/loader-story.3.js.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  return {
+    components: { TodoItem },
+    setup() {
+      return { args, todo: todo };
+    },
+    template: `<TodoItem :todo="todo"/>`,
+  };
+};
+
+SampleStory.loaders = [
```
Comment (shilman) [inline docs/snippets/vue/loader-story.3.js.mdx:23]: Primary.loaders?

File: docs/writing-stories/loaders.md [hunk 4]
```diff
@@ -2,30 +2,36 @@
...
+    'angular/loader-story.mdx.mdx',
     'svelte/loader-story.js.mdx',
+    'svelte/loader-story.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-The loaded data is combined into a `loaded` field on the story context, which is the second argument to a story function. In this example we spread the story's args in first, so they take priority over the static data provided by the loader.
+The response obtained from the remote API call is combined into a `loaded` field on the story context, which is the second argument to a story function. For example, in React, the story's args were spread first to prioritize them over the static data provided by the loader. Other frameworks will not need such requisite.
```
Comment (shilman) [inline docs/writing-stories/loaders.md:34]: > frameworks will not need such requisite.

not sure what this means

**Review by shilman — CHANGES_REQUESTED** [review]
Great job clarifying the documentation! A few typos:

File: docs/writing-stories/loaders.md (hunk 4 above)
Comment (jonniebigodes) [reply docs/writing-stories/loaders.md:34]: I'll clarify it. For instance with Angular we're not spreading args at all. This is only applicable to React.

File: docs/snippets/common/storybook-preview-global-loader.js.mdx (hunk 1 above)
Comment (jonniebigodes) [reply docs/snippets/common/storybook-preview-global-loader.js.mdx]: Fixed with the latest commit. Marking it as solved

File: docs/snippets/vue/loader-story.3.js.mdx (hunk 3 above)
Comment (jonniebigodes) [reply docs/snippets/vue/loader-story.3.js.mdx]: Fixed with the latest commit. Marking this as solved

File: docs/writing-stories/loaders.md (hunk 4 above)
Comment (jonniebigodes) [reply docs/writing-stories/loaders.md]: With the latest commit, I've addressed the typos (thanks for this @shilman ), and added a bit of context on this item of feedback. Let me know if you're ok and we can merge it.

**Review by shilman — APPROVED** [review]
Thanks for the updates! 🙏

**Pull request merged**